dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.json

```diff
@@ -0,0 +1,476 @@
+{
+ "title": "Code.org PL workshop survey library",
+ "pages": [
+  {
+   "name": "page1",
+   "elements": [
+    {
+     "type": "html",
+     "name": "clickTo44",
+     "visible": false,
+     "titleLocation": "hidden",
+     "html": "<h2 style=\"background-color: #ffff00\"> Variable Test</h2>\n<ul>\n<li><span style=\"background-color: #ffff00;\">environment: {environment}</span></li>\n<li><span style=\"background-color: #ffff00;\">userId: {userId}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopId: {workshopId}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopCourse: {workshopCourse}</span></li>\n<li><span style=\"background-color: #ffff00;\">workshopSubject: {workshopSubject}</span></li>\n<li><span style=\"background-color: #ffff00;\">regionalPartnerName: {regionalPartnerName}</span></li>\n<li><span style=\"background-color: #ffff00;\">sessionId: {sessionId}</span></li>\n<li><span style=\"background-color: #ffff00;\">submitRedirect: {submitRedirect}</span></li>\n</ul>"
```

Comment:
variables should be in lowercase with underscore: {workshop_course}, {user_id}, {workshop_subject}, etc.

Reply:
Fixed.